docker/develop/Dockerfile

```diff
@@ -25,16 +25,23 @@ RUN apt-get -y --no-install-recommends install build-essential python-software-p
         # code coverage
         lcov \
         # other
-        wget curl file unzip gdb iputils-ping vim ccache \
+        wget file unzip gdb iputils-ping vim ccache \
         gcovr cppcheck doxygen graphviz graphviz-dev; \
     apt-get -y clean
 
-RUN pip3 install --upgrade pip; \
-    pip3 install flask requests
 
-# install cmake 3.5.1
+# install curl with SSL, 7.58.0
+RUN (cd /tmp; wget  https://curl.haxx.se/download/curl-7.58.0.tar.gz; \
```

Comment:
What's the problem with the previous version?

Reply:
Default CURL from APT is compiled without SSL support, and Hunter (which naturally uses CURL to download binary distributions from github) fails with error.

Reply:
Is there a way to disable SSL in Hunter? It's unlikely that users will have curl with SSL by default.

Reply:
It is possible to disable SSL https://docs.hunter.sh/en/latest/faq/how-to-fix-download-error.html?highlight=SSL but actually, SSL issue is mostly related to docker. If you have a desktop linux version or mac, it is likely to have libssl or libboringssl and libcurl with SSL support.